content/connect/data-types/_index.md

```diff
@@ -0,0 +1,504 @@
+
+Not supported
+
+### geometrycollection {#m-geometrycollection}
+
+Not supported
+
+### json {#m-json}
+
+Target data type (hash): string
+
+Example: `{"pid": 101, "key1": "value1", "key2": "value2"}`
+
+### linestring {#m-linestring}
+
+Not supported
+
+### multilinestring {#m-multilinestring}
+
+Not supported
+
+### multipoint {#m-multipoint}
+
+Not supported
+
+### multipolygon {#m-multipolygon}
+
+Not supported
+
+### polygon {#m-polygon}
+
+Not supported
+
+### set {#m-set}
+
+Target data type (hash): string
+
+Example: A MySQL value of `"1,2,3"` is stored in a target Redis database as  `"1,2,3"`.
+
+## Oracle specific types
+
+Oracle supports several data types beyond the traditional [ANSI data types](#ansi-types), including:
+
+&nbsp; &nbsp;[bfile](#o-bfile), [binary_double](#o-binary-double), [binary_float](#o-binary-float), [clob](#o-clob), [float, real, double precision](#o-float)  
+
+&nbsp; &nbsp;[long_raw](#o-long-raw), [nchar](#o-nchar), [nclob](#o-nclob), [number](#o-number), [nvarchar](#o-nvarchar)  
+
+&nbsp; &nbsp;[raw](#o-raw), [rowid](#o-rowid), [timestamp with tz](#o-timestamp), [urowid](#o-urowid)
+
+### bfile {#o-bfile}
+
+Not supported
+
+### binary_double {#o-binary-double}
+
+Target data type (hash): string
+
+Example: `"1.7E+308"`
+
+### binary_float {#o-binary-float}
+
+Target data type (hash): string
+
+Example: `"3.40282E38"`
+
+### clob {#o-clob}
+
+Target data type (hash): string
+
+### float, real, double precision {#o-float}
+
+Target data type (hash): string  
+
+Example: real-FLOAT(63), double precision - FLOAT(126)
+
+The `decimal.handling.mode` Debezium connector configuration setting affects this.  When set to `double`, an input value of `-3.402E+38` is saved to the target Redis database as  `"-340200000000000000000000000000000000000"`.
+
+### long_raw {#o-long-raw}
+
+Not supported
+
+### nchar {#o-nchar}
+
+Target data type (hash): string supporting Unicode characters  
+
+Example: An Oracle input value of "Peace in Hebrew is שלום" is saved to the Redis database as `"Peace in Hebrew is \xd7\xa9\xd7\x9c\xd7\x95\xd7\x9d"`.  Trailing spaces are added to fill the field value.
+
+### nclob {#o-nclob}
+
+Not supported
+
+### number(p, s) {#o-number}
+
+Target data type (hash): string  
+
+Example: `"10385274000.32"`
+
+### nvarchar {#o-nvarchar}
+
+Target data type (hash): string supporting Unicode characters  
+
+Example: An Oracle input value of "Peace in Hebrew is שלום" is saved to the Redis database as `"Peace in Hebrew is \xd7\xa9\xd7\x9c\xd7\x95\xd7\x9d"`.
+
+### raw {#o-raw}
+
+Not supported
+
+### rowid {#o-rowid}
+
+Target data type (hash): string  
+
+Example: `"AAAR1QAAOAAAACFAAA"`
+
+### timestamp with tz {#o-timestamp}
+
+Target data type (hash): string  
+
+Example: Debezium converts an an Oracle input value of `2021-12-30 14:23:46` to `"2021-12-30T14:23:46+02:00"`.  This is stored in the target Redis database as `"1611878400000"`, which represents the milliseconds since the the Epoch.
```

Comment:
```suggestion
Example: Debezium converts an Oracle input value of `2021-12-30 14:23:46` to `"2021-12-30T14:23:46+02:00"`.  This is stored in the target Redis database as `"1611878400000"`, which represents the milliseconds since the the Epoch.
```

Reply:
Fixed